templates/frontend/layout.html.twig

```diff
@@ -37,6 +37,7 @@
 
 {% block javascripts %}
     <script type="text/javascript" src="{{ asset('assets/frontend/app.js', 'frontend') }}"></script>
+    {{ sonata_block_render_event('sylius.shop.layout.javascripts') }}
```

Comment:
What is `shop` in the realm of Monofony?

Reply:
We could change it but with that we can call the same event for a Sylius Plugin and a Monofony one

Reply:
Yes "shop" term is replaced with "app" on Monofony (AppUser for example, ) but to fit with sylius events for plugin, it's coherent.

Reply:
Maybe we can rename it with `app` and a plugin which need to be compatible with Sylius and Monofony will listen `sylius.shop.layout.javascripts` and `sylius.app.layout.javascripts` ?

Reply:
That certainly would be better imo.

Reply:
maybe frontend and backend to fit with assets?